language/anope.de_DE.po

```diff
@@ -9611,3 +9661,6 @@ msgstr "{Nickname | Kanal}"
 
 msgid "{nick | channel} memo-text"
 msgstr "{Nickname | Kanal} Memo-Text"
+
+#~ msgid "Password authentication required for that command."
+#~ msgstr "Password Authentisierung ist notwendig für diesen Befehl."
```

Comment:
As these lines are obsolete (replaced with *You must be logged into an account to use that command.*) you can remove this.